src/appengine/handlers/cron/project_setup.py

```diff
@@ -527,10 +532,16 @@ def create_project_settings(project, info, service_account):
       oss_fuzz_project.ccs = ccs
       oss_fuzz_project.put()
   else:
+    cpu_weight = (
```

Comment:
I would convert this to a normal `if-else` the one line version is not very readable when it's more than one line.

Reply:
Done!